### PR TITLE
(MAINT) Update CHANGELOG for 0.2.0

### PR DESCRIPTION
**Pull request by adrienthebo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## Release 0.2.0
+
+This is a feature and bugfix release.
+
```
Comment (haus) [inline CHANGELOG.md]: which are the features and which are the bugs?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+* (MODULES-4523) Return expiration of new CA cert for `puppet certregen ca`; previously the expiration of the old CA certificate was was returned.
```
Comment (camlow325) [inline CHANGELOG.md]: s/was was/was

File: CHANGELOG.md (hunk 1 above)
Comment (adrienthebo) [inline CHANGELOG.md]: Good call, I've broken the changes up into sections to match the previous release entry.

File: CHANGELOG.md (hunk 2 above)
Comment (adrienthebo) [inline CHANGELOG.md]: Fixed.

**Comment by adrienthebo** [comment]
I've amended the commit in place since this is a pretty small change.

**Comment by haus** [comment]
Travis is failing with:

```
---> syntax:manifests
---> syntax:templates
---> syntax:hiera:yaml
(WARN) dependencies: Dependency puppetlabs/stdlib has an open ended dependency version requirement 
Warnings found in metadata.json
The command "bundle exec rake release_checks" exited with 1.
```

not sure if we care or not.

**Comment by adrienthebo** [comment]
@camlow325 @haus CI has been fixed; this should be ready for merge.

**Pull request merged**